executor/executor.yml

```diff
@@ -218,4 +226,15 @@
 
 - debug:
     msg: "engine : {{ c_engine_yaml.stdout }}" 
- 
\ No newline at end of file
+ 
```

Comment:
@rahulchheda  From where we are getting `c_engine_yaml`? Do we need a debug message here?